cmd/oras/repository/ls.go

```diff
@@ -25,8 +26,9 @@ import (
 type repositoryOptions struct {
 	option.Remote
 	option.Common
-	hostname string
-	last     string
+	hostname  string
+	namespace string
```

Comment:
A reference can have many [namespaces](https://learn.microsoft.com/azure/container-registry/container-registry-best-practices#repository-namespaces).

Reply:
How is `namespace` typically stored?